### ISSUE TITLE
More button options needed

### ISSUE THREAD

**Issue by angela-crist** [faithmade/faith-builder#29]
Right now the button design options are limited.  There's no way to get buttons with a thick border and transparent background color.  The options that I think would be good to add include:
1. Button border color
2. Button background transparency
3. Button hover transparency

**Comment by garand** [comment]
You're talking about the button module in Faith Builder, correct?

**Comment by angela-crist** [comment]
Yep, you got it Anthony.  This was a suggestion from James when he created the layouts. This is not included in the upcoming milestone so this is very low priority and may be closed after the next milestone because it is not a priority.
